pyempaq/main.py

```diff
@@ -146,14 +155,25 @@ def main():
     parser.add_argument(
         "source", type=pathlib.Path,
         help="The source file (pyempaq.yaml) or the directory where to find it.")
+    parser.add_argument(
+        '-v', '--verbose',
+        help="Show detailed information, typically of interest only when diagnosing problems.",
+        action="store_const", dest="loglevel", const=logging.DEBUG)
+    parser.add_argument(
+        '-q', '--quiet',
+        help="Only events of WARNING level and above will be tracked",
+        action="store_const", dest="loglevel", const=logging.WARNING)
     args = parser.parse_args()
+
+    if not args.loglevel is None:
```

Comment:
cuando se usa el `is` se recomienda negarlo directamente, por legibilidad: `if args.loglevel is not None`